Page branding

MelanyCristy opened the pull request:
Me and Pinelopi changed most of the style. Still a little things to on work on but so far we can say it's done the first step.

MelanyCristy committed:
Add new branding and logo

MelanyCristy committed:
Add logo and change background color

SaraGarabedian commented on frontend/package-lock.json:
```diff
@@ -9,28 +9,28 @@
-        "resolve": "1.12.0",
-        "semver": "5.7.1",
-        "source-map": "0.5.7"
+        "@babel/code-frame": "^7.5.5",
```
What does this change do?
Why are the versions changed?

krismutalapova reviewed (approved):
LGTM

SaraGarabedian commented on frontend/src/components/home/HomePage.js:
```diff
@@ -5,14 +5,14 @@ class HomePage extends Component {
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
+                        <li><a href=""></a></li>
```
Why keep the tags for the ul when emptied?

SaraGarabedian commented on frontend/src/index.css:
```diff
@@ -1,10 +1,11 @@
 body {
-  margin: 0;
+   margin: 0;
```
Unnecessary formatting, should be removed

SaraGarabedian reviewed (commented):
Could you rephrase your commit, so it says what it does, such as:
Instead of "Me and Pinelopi changed most of the style. Still a little things to on work on but so far we can say it's done the first step."
Something like:
"Added a new logo, changed background, updated versions, etc."

MelanyCristy committed:
Delete unnecesary code

MelanyCristy commented:
> Could you rephrase your commit, so it says what it does, such as:
> Instead of "Me and Pinelopi changed most of the style. Still a little things to on work on but so far we can say it's done the first step."
> Something like:
> "Added a new logo, changed background, updated versions, etc."

**This is the other way around. We did as you mentioned, the first message is just a pull request description. The commit messages are the ones with the description of our commits**

MelanyCristy committed:
Unnecesary space removed
